src_c/display.c

```diff
@@ -2868,27 +2869,35 @@ pg_get_screensaver_enabled(PyObject *self) {
 #if IS_SDLv2
     return PyBool_FromLong(SDL_IsScreenSaverEnabled() == SDL_TRUE);
 #else /* IS_SDLv1*/
-    return RAISE(PyExc_TypeError, "get_screensaver_enabled requires SDLv2");
+    return PyBool_FromLong(_allow_screensaver);
 #endif /* IS_SDLv1*/
 }
 
 static PyObject *
-pg_set_screensaver_enabled(PyObject *self, PyObject* args) {
-    int val;
-    if (!PyArg_ParseTuple(args, "i", &val))
+pg_set_screensaver_enabled(PyObject *self, PyObject *arg, PyObject *kwds) {
+    int val = 0;
+    static char *keywords[] = {"value", NULL};
+
+    if (!PyArg_ParseTupleAndKeywords(arg, kwds, "i", keywords, val)) {
```

Comment:
Does this mean that `set_screensaver_enabled()` works (i.e. with no arguments)?  If so, I think "False" is a weird default for this function.

Reply:
I'm not sure I considered what to do with no value passed in really.


What should it do when called like `set_allow_screensaver()`? Reading the call like that, it reads like the intention is 'allow screensaver'. I guess the default should be True?

Reply:
If it accepts no arguments, I would assume the default was true.  I am a bit conflicted on it accepting no parameters (among other because I think `allow_screensaver([bool])` is a better variant for an argumentless variant - i.e. without set - but it does not follow our existing conventions and is therefore confusing for a different reason).